models/cut_model.py

```diff
@@ -244,15 +244,14 @@ def compute_G_loss(self):
 
     def calculate_NCE_loss(self, src, tgt):
         n_layers = len(self.nce_layers)
-        feat_q = self.netG(tgt, self.nce_layers, encode_only=True)
+        feat_q = self.netG(tgt, self.nce_layers, True)
```

Comment:
maybe we can leave the `encode_only=` as more readable.

Reply:
I wish but unfortunately we can't, it's a fix because dataparallel doesn't handle the same way args and kwargs